src/repl/history.rs

```diff
@@ -68,8 +68,10 @@ impl History {
 
     pub fn load() -> Self {
         let mut history = History(None, vec![]);
-        if Path::new("/home/nixpulvis/.oursh_history").exists() {
-            let mut f = File::open("/home/nixpulvis/.oursh_history")
+        let home_dir = dirs::home_dir().unwrap();
```

Comment:
```suggestion
        let home = env::var("HOME").expect("HOME variable not set");
```

Reply:
The documentation for the deprecated `std::env::home_dir` does say to do it your way, however that's if you want the same functionality exactly. Since we don't care about windows, I don't think we need this.

Reply:
https://github.com/nixpulvis/oursh/blob/master/src/program/builtin.rs#L46

Reply:
Makes sense; changed!